gslib/commands/pap.py

```diff
@@ -30,7 +30,7 @@
 from gslib.utils.constants import NO_MAX
 
 _SET_SYNOPSIS = """
-  gsutil pap set (enforced|unspecified) gs://<bucket_name>...
+  gsutil pap set (enforced|inherited) gs://<bucket_name>...
```

Comment:
Oh, I have made similar changes in another PR, which does this in addition to removing unspecified from the code itself.  https://github.com/GoogleCloudPlatform/gsutil/pull/1445

Reply:
This can be merged btw, my comment was more of an FYI.